tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -1455,11 +1455,14 @@ class AnnotationExtendsAnnotationTargetAll extends AnnotationTargetAll
 {
 }
 
-/** @Annotation */
-class Name extends Annotation {
-    public $foo;
+if(!class_exists('\Doctrine\Tests\Common\Annotations\Name')) {
```

Comment:
Use `::class` here

Reply:
This seems weird... Why the `if` in first place?

Reply:
Please commend and run the test on class: AnnotationReaderTest you will see it fail because it depend on \Doctrine\Tests\Common\Annotations\Name::class

Reply:
Any way to use `@runInSeparateProcess` instead?